force-app/main/default/classes/RD2_ERecurringDonationsSelector.cls

```diff
@@ -50,29 +53,38 @@ public inherited sharing class RD2_ERecurringDonationsSelector {
      * @description Returns the Recurring Donation records related to the Experience Site user.
      * @return List<npe03__Recurring_Donation__c>
      */
-    public List<npe03__Recurring_Donation__c> getRecurringDonations() {
+    public List<npe03__Recurring_Donation__c> getRecurringDonations(String elevateFilter) {
 
 		Id currentContactId = this.currentContactId;
 		List<npe03__Recurring_Donation__c> recurringDonations = new List<npe03__Recurring_Donation__c>();
 		if(currentContactId != NULL) {
-			recurringDonations = [  SELECT npe03__Amount__c,
-										npe03__Installment_Period__c,
-										InstallmentFrequency__c,
-										Day_of_Month__c,
-										Status__c,
-										CommitmentId__c,
-										PaymentMethod__c,
-										CardLast4__c,
-										CardExpirationYear__c,
-										CardExpirationMonth__c,
-										ACH_Last_4__c,
-										npe03__Last_Payment_Date__c,
-										npe03__Next_Payment_Date__c
-									FROM npe03__Recurring_Donation__c
-									WHERE npe03__Contact__c = :currentContactId
-                  					AND RecurringType__c IN (:RD2_Constants.RECURRING_TYPE_OPEN, :RD2_Constants.RECURRING_TYPE_FIXED) 
-									WITH SECURITY_ENFORCED 
-									ORDER BY npe03__Next_Payment_Date__c DESC NULLS LAST];
+			String queryString = 'SELECT npe03__Amount__c, '
```

Comment:
For indentations in this file - and any file in the repo - please use 4 spaces for indentations and not tabs. Looking at this PR, it's clear the files now have a mix of spaces and tabs for indentation. You should be able to use your IDE to fix the indentations and the recommit. Please ensure that all of your team members are also using spaces and not tabs. Consistency is critical in a product of this size.

Reply:
Thanks Micheal, I'll have this into account. Let me fix this.